config/samples/v1_prefectworkpool_external_prefect_server.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: prefect.io/v1
+kind: PrefectWorkPool
+metadata:
+  labels:
+    app.kubernetes.io/name: prefect-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: external-pool
+spec:
+  type: process
+  server:
+    remoteApiUrl: https://my-server.example.com
```

Comment:
I noticed that we apply the `/api` suffix if one isn't provided, so I think it's fine to leave it out of the example - but I noticed that Prefect Cloud uses an `api` subdomain too: `api.prefect.cloud/api`. I'm guessing that's just a cloud-specific thing 🤔

Reply:
Yep that's specific to Prefect Cloud, but folks who are hosting it themselves can use any domain they like (but it's always a subpath of `/api`)